config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   get 'expenses/create'
 
-  devise_for :users
+  devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
```

Comment:
I'm not convinced that having this controller in a folder named `users` is the best way to go. I would move it out, rename the controller and change the pointer here in `routes.rb`

Reply:
Ok, I'll give this a go 👍